src/domain/audio/InboundAudioStream.ts

```diff
@@ -140,7 +140,11 @@ class InboundAudioStream {
             console.debug("$$$$$$$ Ramp up volume of this message.");
 
             // TODO: Ramp up volume of this message - scale samples.
-
+            const audioBuffer = (info as MixedAudioDetails).audioBuffer;
+            for (let i = 0, length = audioBuffer.byteLength; i < length; i += 2) {
+                const scale = i / (length-2);
```

Comment:
Change to:
```
const scale = i / length;
```
The first sample in the next packet will be 100% and removing the "- 2" saves some CPU.